userguide/system.rst

```diff
@@ -2248,8 +2252,8 @@ To generate a support ticket, fill in the fields:
   level. Choices are *Inquiry*, *Loss of Functionality*, or
   *Total Down*.
 
-* **Attach Debug Info** it is recommended to leave this box checked so
-  that an overview of the system’s hardware, build string, and
+* **Attach Debug Info** it is recommended to leave option selected so
```

Comment:
"It is recommended"

leaving this option selected is recommended so an overview....